core/arch/arm/mm/tee_pager.c

```diff
@@ -770,4 +809,38 @@ void tee_pager_add_pages(vaddr_t vaddr, size_t npages, bool unmap)
 		core_tlb_maintenance(TLBINV_UNIFIEDTLB, 0);
 	}
 }
+
+void tee_pager_release_zi(vaddr_t vaddr, size_t size)
+{
+	bool unmaped = false;
+	uint32_t exceptions = thread_mask_exceptions(THREAD_EXCP_ALL);
```

Comment:
When releasing pages, we are calling this function directly, not through page fault. This is why exceptions are masked.

When we request pages, we only indicate that in case of page fault, then this page can be loaded. We do not change the mapping.